offline-build/builder

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+image=cockroachdb/offline-docs
```

Comment:
Does this docker hub repo exist already?

Reply:
No -- I think I need you to create it.
Also, not sure about deployment process for this yet. This is copy-pasta from elsewhere.